h2o-py/tests/testdir_misc/pyunit_ice_plot.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function
+
+import sys
+
+sys.path.insert(1,"../../")
+import matplotlib
+matplotlib.use("Agg")  # remove warning from python2 (missing TKinter)
+import h2o
+import matplotlib.pyplot
+from tests import pyunit_utils
+from h2o.estimators import *
+
+
+def test_display_mode():
+    import matplotlib
```

Comment:
nit: This is I guess also the unnecessary import as in the other PR.

Reply:
f3437b8